html/hidden3.html

```diff
@@ -21,28 +21,32 @@ <h3 class="bigTitle" style="text-align: center; color:black; font-size: 30px; ma
   <form >
     <div class="form-outline" style="margin-top: 20px">
       <label class="form-label" for="form12" style="font-weight: normal; font-size: 16px;">First Name:</label>
-      <input style="height: 40px;" type="text" placeholder="Enter Last Name" id="form12" class="form-control" name="firstName" required/>
+      <input style="height: 40px;" type="text" placeholder="Enter First Name" id="firstName" class="form-control" name="firstName" required/>
     </div>
       <div class="form-outline" style="margin-top: 20px">
         <label class="form-label" for="form12" style="font-weight: normal; font-size: 16px;">Last Name:</label>
-        <input style="height: 40px;" type="text" placeholder="Enter Last Name" id="form12" class="form-control" name="lastName" required/>
+        <input style="height: 40px;" type="text" placeholder="Enter Last Name" id="lastName" class="form-control" name="lastName" required/>
       </div>
       <div class="form-outline" style="margin-top: 20px">
         <label class="form-label" for="form12" style="font-weight: normal; font-size: 16px;">Email Id:</label>
-        <input style="height: 40px;" type="text" placeholder="Enter Email Id" id="form12" class="form-control" required/>
+        <input style="height: 40px;" type="text" placeholder="Enter Email Id" id="emailAdress" class="form-control" required/>
       </div>
       <div class="form-outline" style="margin-top: 20px">
         <label class="form-label" for="form12" style="font-weight: normal; font-size: 16px;">Contact No:</label>
-        <input style="height: 40px;" type="text" placeholder="Enter Contact Number" id="form12" class="form-control" required/>
+        <input style="height: 40px;" type="text" placeholder="Enter Contact Number" id="contactNumber" class="form-control" required/>
       </div>
       <div class="form-check" style="margin-top: 20px; margin-left: 2px;">
         <input class="form-check-input" type="checkbox" value="" id="flexCheckDefault" />
         <label class="form-check-label" for="flexCheckDefault" style="margin-left: 7px; font-weight: normal; font-size: 17px;">Remember Me</label>
       </div>
     </div>
+    <div>
+      <button class="btn btn-primary" id="btn" type="button"
+      style="font-weight: normal; margin:25px; margin-left: 25px; background-color: rgb(15, 180, 15); color:rgb(55, 52, 52)" onclick="apiCallOnButtonClick()">Fill with random data</button>
```

Comment:
Normally we use function names that describes better what it is doing, like "fillFormWithRandomData()". It makes it easier to understand.

Reply:
i couldn't come up with a good name but this one is better indeed, changed it!